products/sqlite.md

```diff
@@ -0,0 +1,333 @@
+---
+title: SQLite
+category: db
+iconSlug: sqlite
+permalink: /sqlite
+releasePolicyLink: https://www.sqlite.org/news.html
+changelogTemplate: https://sqlite.org/releaselog/__LATEST__/.html
+activeSupportColumn: false
+eolColumn: Support Status
+versionCommand: sqlite3 --version
+releaseDateColumn: true
+auto:
+-   git: https://github.com/sqlite/sqlite.git
```

Comment:
Given the tag name (such as `version-3.40.1`) a regex is needed.

Reply:
All tags are not available on GH : what should be done ?

Reply:
What is the range of available tags ?

Reply:
> Are multiple minor versions of the same major supported in parallel ? If no, 3 cycles should be collapsed.

I could first intent a merge in single majors versions @marcwrobel :grey_question:

Reply:
Yes, that would be better to have only major version cycles IMHO.

Reply:
Regex added.